modules/cluster/src/main/java/org/apache/fluo/cluster/runner/AppRunner.java

```diff
@@ -234,7 +234,7 @@ public String getRowPrefix() {
 
     public ScanUtil.ScanOpts getScanOpts() {
       return new ScanUtil.ScanOpts(startRow, endRow, columns, exactRow, rowPrefix, help,
-          hexEncNonAscii, scanAccumuloTable, false);
+          hexEncNonAscii, scanAccumuloTable, false, false);
```

Comment:
boolean arguments like this are kind of a bad idea; should probably use enums. have no idea what these booleans represent at a glance, and multiple sequential params of the same time are prone to bugs.

Reply:
I updated to use EnumSet and a new Enum.